drivers/iio/light/apds9960.c

```diff
@@ -1018,6 +1039,10 @@ static int apds9960_probe(struct i2c_client *client,
 		return PTR_ERR(data->regmap);
 	}
 
+	if (device_property_read_u32(&client->dev, "proximity-near-level", &data->near_level) < 0) {
+		data->near_level = 0;
+	}
```

Comment:
Where do you get this attribute from? I can't see that anywhere in the DSDT/acpidump. The only reference to it that I can find is in some device-tree files for another sensor. At this point, I have to ask: Do we even need that? If so, for what?

As far as I can tell, you currently a) don't have any way to get a sane value for this attribute (so on the SB3 it will always be zero) and b) probably don't have any user-space tool/library that depends on this, as you should have noticed the typo above if you had. So based on that, it doesn't make any sense to me why we should currently add this.

Also, in Linux coding style it is preferred to omit braces here and the line exceeds 100 columns. FYI, the `checkpatch.pl` script of the kernel can spot these things for you, especially if you run it with the `--strict` option.